ethers-signers/src/trezor/types.rs

```diff
@@ -72,9 +72,20 @@ impl TrezorTransaction {
     }
 
     pub fn load(tx: &TypedTransaction) -> Result<Self, TrezorError> {
-        let to: String = match tx.to().unwrap_or(&NameOrAddress::Address(H160::from(&[0; 20]))) {
-            NameOrAddress::Name(_) => unimplemented!(),
-            NameOrAddress::Address(value) => format!("0x{}", hex::encode(value)),
+        let to: String = match tx.to() {
+            Some(v) => match v {
+                NameOrAddress::Name(value) => {
+                    // Contract Creation
+                    if value.is_empty() {
+                        "".to_string()
+                    } else {
+                        unimplemented!()
+                    }
```

Comment:
think we should just be erroring here if a user passes an ENS name instead of silently letting it through in the empty case or panicking if it's not empty?